GOVERNANCE_DOC.md

```diff
@@ -0,0 +1,597 @@
+                // i.e. sender == validator
+
+                retrieve initialVotingPower from InitVotingPowerList using txVote.ValidatorPubKey
+                
+                
+                if exists(MinusesList[txVote.ValidatorPubKey]) then
+                  // a minus exists for this validator's PubKey, decrease vote of validator by minus
+
+                  proposal.Votes['txVote.Option'] += (initialVotingPower - MinusesList[txVote.ValidatorPubKey])
+
+                else
+                  // a minus does not exist for this validator's PubKey, validator votes with full voting power
+
+                  proposal.Votes['txVote.Option'] += initialVotingPower
+                  
+              if (proposal.Category AND proposal.Votes['Yes']/InitTotalVotingPower >= 2/3)
+                // after vote is counted, if proposal is urgent and special condition is met
+                // remove proposalID from ProposalProcessingQueue
+                
+                remove txVote.ProposalID from ProposalProcessingQueue
+                Rearrange ProposalProcessingQueue
+                ProposalProcessingQueueEnd--
+
+              
+```
+
+
+## Future improvements (not in scope for MVP)
+
+The current documentation only describes the minimum viable product for the governance module. Future improvements may include:
+
+- **`BountyProposals`:** If accepted, a `BountyProposal` creates an open bounty. The `BountyProposal` specifies how many Atoms will be given upon completion. These Atoms will be taken from the `reserve pool`. After a `BountyProposal` is accepted by governance, anybody can submit a `SoftwareUpgradeProposal` with the code to claim the bounty. Note that once a `BountyProposal` is accepted, the corresponding funds in the `reserve pool` are locked so that payment can always be honored. In order to link a `SoftwareUpgradeProposal` to an open bounty, the submitter of the `SoftwareUpgradeProposal` will use the `Proposal.LinkedProposal` attribute. If a `SoftwareUpgradeProposal` linked to an open bounty is accepted by governance, the funds that were reserved are automatically transferred to the submitter.
+- **Complex delegation:** Delegators could choose other representatives than their validators. Ultimately, the chain of representatives would always end up to a validator, but delegators could inherit the vote of their chosen representative before they inherit the vote of their validator. In other words, they would only inherit the vote of their validator if their other appointed representative did not vote.
+- **`ParameterProposals` and `WhitelistProposals`:** These proposals would automatically change pre-defined parameters and whitelists. Upon acceptance, these proposals would not require validators to do the signal and switch process.
+- **Better process for proposal review:** There would be two parts to `proposal.Deposit`, one for anti-spam (same as in MVP) and an other one to reward third party auditors.
```

Comment:
Also more complex voting styles? such as ranked voting??? <- big fan of this idea

Reply:
Ranked voting is cool but not near future. These are things next in line just after MVP